### PR TITLE
RSDK-2252 - Unable to run cartogropher through RDK

### PR DESCRIPTION
**Pull request by kkufieta**
Ticket: https://viam.atlassian.net/browse/RSDK-2252

rdk side: https://github.com/viamrobotics/rdk/pull/2021

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -130,3 +130,38 @@ func (config *AttrConfig) SetOptionalParameters(defaultPort string, defaultDataR
...
+		logger.Info("setting slam system to localization mode")
+	}
+
+	useLiveData, err := DetermineUseLiveData(logger, config.UseLiveData, config.Sensors)
+	if err != nil {
+		return "", 0, 0, false, false, err
+	}
+
+	deleteProcessedData := DetermineDeleteProcessedData(logger, config.DeleteProcessedData, useLiveData)
+	config.DeleteProcessedData = &deleteProcessedData
```
Comment (tessavitabile) [inline config/config.go:164]: Should this line be removed?

**Review by tessavitabile — APPROVED** [review]
LGTM mod comment

**Review by zaporter-work — APPROVED** [review]
This code will be changed and refactored in the future. Ticket: https://viam.atlassian.net/browse/RSDK-2267

File: config/config.go (hunk 1 above)
Comment (nicksanford) [reply config/config.go:164]: It didn't trigger reconfiguration but yes it should be removed.

**Review by nicksanford — CHANGES_REQUESTED** [review]

File: config/config.go (hunk 1 above)
Comment (kkufieta) [reply config/config.go]: yes, done.

**Review by nicksanford — APPROVED** [review]

**Pull request merged**